lib/src/utils/serialization_utils.dart

```diff
@@ -75,9 +75,11 @@ class SerializationUtils {
   }
 
   static Uint8List serializedUint64(BigInt value) {
-    if (value.isNegative) {
-      throw ValidationException("writeSerializedUint64 - Value is negative");
-    } else if (value.bitLength > 64) {
+    //TODO: Should check because there are cases where values could be negative
+    // if (value.isNegative) {
+    //   throw ValidationException("writeSerializedUint64 - Value is negative");
+    // } 
+    if (value.bitLength > 64) {
       throw ValidationException("writeSerializedUint64 - Value is too large");
```

Comment:
If this happens, you are using the wrong function or there's another mistake. As the name says, this is for `Uint64` only. `U` comes from unsigned. If it can be negative, you need some `serializedInt64` not `serializedUInt64`.

Reply:
done, it was meant for ParsedAssetToken token.amount value